### DISCUSSION TITLE
Is it possible to reset a single !M metadata value?

### DISCUSSION THREAD

**Discussion by cprima** [gopro/labs#357] [Q&A] [answered]
Let's say I did set "permanent across reboots" !MEXPX=150,
but after a few days would like to un-set this value.

Would I need to set !MEXPX=30,
or is there a reset method?

I know that !RESET will reset all metadata values and this is certainly a possibility for me,
but maybe I just missed an alternative.

The usecase is:
I want to quickly change with a QR code
- from in-camera stabilized
- to stabilization in post

That requires me (un)setting
- EXPX
- EXPN
- lens
- Hypersmooth
- and potentially some more

**Comment by dnewman-gpsw** [comment, marked as answer]
!MEXPX=0  or any !Mabcd=0 disables the function.
